ReactiveCocoa/Swift/Property.swift

```diff
@@ -15,6 +15,37 @@ public protocol PropertyType {
 	var signal: Signal<Value, NoError> { get }
 }
 
+extension PropertyType {
+	/// Maps each value in the property to a new value.
+	@warn_unused_result
+	public func map<U>(transform: Value -> U) -> AnyProperty<U> {
+		return AnyProperty(
+			initialValue: transform(value),
+			signal: signal.map(transform)
```

Comment:
I suppose you pick the `signal` overload because replaying twice the current value doesn't seem a good thing. As an alternative, we may consider moving `withValue(action:)` from `MutablePropertyType` to `PropertyType`, so these compositing methods can be synchronised as needed.